internal/azure/advisor/aks_recommendations.go

```diff
@@ -0,0 +1,474 @@
+package advisor
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/Azure/aks-mcp/internal/az"
+	"github.com/Azure/aks-mcp/internal/config"
+)
+
+// HandleAdvisorRecommendation is the main handler for Azure Advisor recommendation operations
+func HandleAdvisorRecommendation(params map[string]interface{}, cfg *config.ConfigData) (string, error) {
```

Comment:
Since we are using az cli for advisor, I feel like we don't need a separate handler. Putting the logic in az/internal/registry.go would be an easier and more clear.